src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Summarize.cs

```diff
@@ -205,8 +205,18 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
         }
 
         public override ParamIdentifierStatus GetIdentifierParamStatus(TexlNode node, Features features, int index)
-        {
-            return index > 0 && node is not AsNode ? ParamIdentifierStatus.PossiblyIdentifier : ParamIdentifierStatus.NeverIdentifier;
+        {

```

Comment:
Can we add a test that would fail without this change?

Reply:
yes, @anderson-joyle  is looking at that.